operations/route_handler.rb

```diff
@@ -0,0 +1,15 @@
+module RouteHandler
+  def self.build_route_view_object(route, params)
+    {
```

Comment:
Take a look at the comments for `Query` module & use classes instead of modules for now, it's more semantically correct considering there's nothing shareable here